packages/sources/css/src/components/checkbox/src/index.css

```diff
@@ -0,0 +1,88 @@
+/* Design tokens */
+@import '@vtmn/css-design-tokens/src/colors';
+@import '@vtmn/css-design-tokens/src/spacings';
+@import '@vtmn/css-design-tokens/src/typography';
+
+.vtmn-checkbox[type='checkbox'] {
+  left: 0;
+  opacity: 0.01;
+  box-sizing: border-box;
+}
+
+.vtmn-checkbox[type='checkbox'] + label {
+  position: relative;
+  font-size: var(--vtmn-typo_text-2-font-size);
+  line-height: var(--vtmn-typo_text-2-line-height);
+  cursor: pointer;
+  user-select: none;
+  left: -24px;
+  padding: 8px 8px 8px 28px;
+}
+
+.vtmn-checkbox[type='checkbox'] + label::before {
+  content: '';
+  position: absolute;
+  left: 1px;
+  top: 7px;
+  width: 18px;
+  height: 18px;
+  border-width: 2px;
+  border-style: solid;
+  border-radius: 4px;
+  transition: all 0.275s;
+}
+
+.vtmn-checkbox[type='checkbox']:not(:checked) + label::before {
+  border-color: var(--vtmn-color_grey-light-1);
+}
+
+.vtmn-checkbox[type='checkbox']:not(:checked):not(:disabled):hover
+  + label::before {
+  box-shadow: 0 0 0 7px rgba(0, 16, 24, 0.05);
```

Comment:
We can use the color-mod method to keep our vitamin color variables : 
box-shadow: 0 0 0 rem(7px) color-mod(var(--vtmn-color_grey-dark-1) alpha(5%));